### DISCUSSION TITLE
How to configure kyuubi for cdh6.0 hue?

### DISCUSSION THREAD

**Discussion by 2416210017** [apache/kyuubi#4131] [Q&A]
My hue configuration is as follows. Please refer to the official website documentation. What is the wrong configuration? The options of spark sql cannot be queried in the hue

![image](https://user-images.githubusercontent.com/52597892/211264336-a81c5f3e-c252-4307-9b0f-6a0532cbb974.png)

```bash

[hbase]
hbase_conf_dir=/etc/hbase/conf
thrift_transport=buffered

[desktop]
ldap_username=hive
ldap_password=@Abk!#
use_cherrypy_server=true
app_blacklist=zookeeper,hbase,search,sqoop,security
use_new_editor=true

[beeswax]
hive_server_host=10.9.251.228
hive_server_port=10002
download_row_limit=10000

[notebook]
show_notebooks=true
enable_external_statements=true
enable_batch_execute=true

[impala]
server_host=10.9.251.228
server_port=21052
close_queries=true

[[interpreters]]
[[[sparksql]]]
name=Spark SQL
interface=hiveserver2

[[[hive]]]
name=Hive
interface=hiveserver2

[spark]
sql_server_host=10.10.214.172
sql_server_port=10009
```

![image](https://user-images.githubusercontent.com/52597892/211444911-a4ad553e-3ee8-4007-9814-a0e9ec525ef5.png)

![image](https://user-images.githubusercontent.com/52597892/211264808-545e26ea-0946-4c5b-a043-8f06167b7c81.png)

**Comment by pan3793** [comment]
Do you find clues in the HUE server logs? IIRC, when the "Editor" shows all options, it indicates that the configuration is wrong(e.g. syntax error).

**Reply by 2416210017** [reply]
The debugging has passed, thank you!
---------------------
In addition, I want to ask whether Hue can configure the flink engine of Kyuubi?

**Comment by JonathanRin** [comment]
Hello @2416210017 ,
Is it possible to connect HUE to kyuubi which have kerberos enable on hdfs files used by kyuubi databases ?
Thank you.

**Reply by 2416210017** [reply]
Hello, we have not enabled hdfs kerberos, which makes it easier for hue to adapt to kyuubi.
![image](https://github.com/apache/kyuubi/assets/52597892/3f721eb4-5f00-4d49-b83a-2bb0bbd8d11f)

**Reply by JonathanRin** [reply]
Hello @2416210017,
But we have currently Kerberized hdfs cluster so we have issue to querying it.
I have tested with hue hive notebook without success.
I can post the error

**Reply by JonathanRin** [reply]
![Uploading image.png…]()

**Reply by JonathanRin** [reply]
![Uploading image.png…]()
